language-plutus-core/test/Evaluation/Machines/Budget/Examples/InterList/FoldrInterList.plc.golden

```diff
@@ -0,0 +1,8 @@
+((Right (/\(a0 :: *) -> /\(b0 :: *) -> /\(r :: *) -> \(f : a0 -> b0 -> r -> r) -> \(z : r) -> (/\(a :: *) -> /\(b :: *) -> \(f : (a -> b) -> a -> b) -> (/\(a :: *) -> \(s : (\(a :: *) -> ifix (\(self :: * -> *) -> \(a :: *) -> self a -> a) a) a) -> unwrap s s) {a -> b} (iwrap (\(self :: * -> *) -> \(a :: *) -> self a -> a) (a -> b) (\(s : (\(a :: *) -> ifix (\(self :: * -> *) -> \(a :: *) -> self a -> a) a) (a -> b)) -> \(x : a) -> f ((/\(a :: *) -> \(s : (\(a :: *) -> ifix (\(self :: * -> *) -> \(a :: *) -> self a -> a) a) a) -> unwrap s s) {a -> b} s) x))) {all (a :: *). a -> a} {all (a :: *). all (b :: *). (a -> b -> r -> r) -> (\(a :: *) -> \(b :: *) -> ifix (\(rec :: ((* -> * -> *) -> *) -> *) -> \(spine :: (* -> * -> *) -> *) -> spine ((\(interlist :: * -> * -> *) -> \(a :: *) -> \(b :: *) -> all (r :: *). r -> (a -> b -> interlist b a -> r) -> r) (\(a :: *) -> \(b :: *) -> rec (\(dat :: * -> * -> *) -> dat a b)))) (\(dat :: * -> * -> *) -> dat a b)) a b -> r} (\(rec : (all (a :: *). a -> a) -> (all (a :: *). all (b :: *). (a -> b -> r -> r) -> (\(a :: *) -> \(b :: *) -> ifix (\(rec :: ((* -> * -> *) -> *) -> *) -> \(spine :: (* -> * -> *) -> *) -> spine ((\(interlist :: * -> * -> *) -> \(a :: *) -> \(b :: *) -> all (r :: *). r -> (a -> b -> interlist b a -> r) -> r) (\(a :: *) -> \(b :: *) -> rec (\(dat :: * -> * -> *) -> dat a b)))) (\(dat :: * -> * -> *) -> dat a b)) a b -> r)) -> \(u : all (a :: *). a -> a) -> /\(a :: *) -> /\(b :: *) -> \(f' : a -> b -> r -> r) -> \(xs : (\(a :: *) -> \(b :: *) -> ifix (\(rec :: ((* -> * -> *) -> *) -> *) -> \(spine :: (* -> * -> *) -> *) -> spine ((\(interlist :: * -> * -> *) -> \(a :: *) -> \(b :: *) -> all (r :: *). r -> (a -> b -> interlist b a -> r) -> r) (\(a :: *) -> \(b :: *) -> rec (\(dat :: * -> * -> *) -> dat a b)))) (\(dat :: * -> * -> *) -> dat a b)) a b) -> unwrap xs {r} z (\(x : a) -> \(y : b) -> \(xs' : (\(a :: *) -> \(b :: *) -> ifix (\(rec :: ((* -> * -> *) -> *) -> *) -> \(spine :: (* -> * -> *) -> *) -> spine ((\(interlist :: * -> * -> *) -> \(a :: *) -> \(b :: *) -> all (r :: *). r -> (a -> b -> interlist b a -> r) -> r) (\(a :: *) -> \(b :: *) -> rec (\(dat :: * -> * -> *) -> dat a b)))) (\(dat :: * -> * -> *) -> dat a b)) b a) -> f' x y (rec (/\(a :: *) -> \(x : a) -> x) {b} {a} (\(y' : b) -> \(x' : a) -> f' x' y') xs'))) (/\(a :: *) -> \(x : a) -> x) {a0} {b0}))
+, ({ tally: ({ cpu: ({ })
+| mem: ({ })
```

Comment:
Is this right? Shouldn't there be at least the initial memory usage of the term?

Also, I think there's not much point doing this for all the terms that don't compute, tbh.